InnerEye/ML/SSL/datamodules_and_datasets/transforms_utils.py

```diff
@@ -0,0 +1,280 @@
+#  ------------------------------------------------------------------------------------------
+#  Copyright (c) Microsoft Corporation. All rights reserved.
+#  Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+#  ------------------------------------------------------------------------------------------
+import random
+from collections import Callable
+from typing import Any, List, Tuple
+
+import PIL
+import numpy as np
+import torch
+import torchvision
+from pl_bolts.models.self_supervised.simclr import SimCLREvalDataTransform, SimCLRTrainDataTransform
+from scipy.ndimage import gaussian_filter, map_coordinates
+from torchvision.transforms import ToTensor
+from yacs.config import CfgNode
+
+
+class BaseTransform:
```

Comment:
Do we have a test for these transforms?

e.g. a single random 2D images and we call each function one by one with a fixed config?

Reply:
We are testing the complete pipeline in the test_data_modules tests. However,, I could add additional tests for the custom augmentations i.e. where we don't use built-in torchvision transforms